libphal/libphal.H

```diff
@@ -271,6 +271,15 @@ void putCFAM(struct pdbg_target *proc, const uint32_t addr, const uint32_t val);
  */
 bool isSbeVitalAttnActive(struct pdbg_target *proc);
 
+/**
+ * @brief Check if Hostboot has completed and the control transistioned to Host/PHYP
+ *
+ * @return true when we have moved to PHYP
+ * @return false if there is any error in reading the scom address, 
```

Comment:
This statement looks a bit confusing to me. We name the method as `hasControlTransitionedToHost` but in the method's description we are saying `consider control still in hostboot`. Am I missing something here?

Reply:
Got the point. Sorry. No need to pay attention to this comment anymore.